Samples/auth/Outlook-Add-in-SSO-events/README.md

```diff
@@ -0,0 +1,118 @@
+
+## Configure the Sample
+
+Before you run the sample, you'll need to do a few things to make it work properly.
+
+- In Visual Studio Code (or editor of your choice) , open the root folder for this sample.
+
+### Update manifest.xml
+
+1. Open the **manifest.xml** file.
+1. Find the `<WebApplicationInfo>` section near the bottom of the manifest. Then replace the `Enter_client_ID_here` value, in both places where it appears, with the application ID you generated as part of the app registration process.
+
+>Note: Make sure that the port number in the `Resource` element matches the port used by your project. It should also match the port you used when registering the application.
+
+### Update .ENV
+
+1. Open the **.ENV** file.
+1. Replace the `Enter_client_ID_here` placeholder value with the application ID you generated as part of the app registration process.
+1. Replace the `Enter_client_secret_here` placeholder value with the client secret you generated as part of the app registration process.
+
+## Provide user consent to the app
+
+If you want to try the add-in using a different tenant than the one where you registered the app, you need to do this step.
+
+You have two choices for providing consent:
+
+- All users. Use an administrator account and consent once for all users in your Office 365 tenant
+- Single user. Use any account to consent for just that user
+
+### Provide admin consent for all users
+
+If you have access to a tenant administrator account, this method allows you to provide consent for all users in your organization, which can be convenient if you have multiple developers that need to develop and test your add-in.
+
+1. Browse to `https://login.microsoftonline.com/common/adminconsent?client_id={application_ID}&state=12345`, where `{application_ID}` is the application ID shown in your app registration.
+1. Sign in with your administrator account.
+1. Review the permissions and click **Accept**.
+
+The browser will attempt to redirect back to your app, which may not be running. You might see a "this site cannot be reached" error after clicking **Accept**. This is OK, the consent was still recorded.
+
+### Provide consent for a single user
+
+If you don't have access to a tenant administrator account, or you just want to limit consent to a few users, this method allows you to provide consent for a single user.
+
+1. Browse to `https://login.microsoftonline.com/common/oauth2/authorize?client_id={application_ID}&state=12345&response_type=code`, where `{application_ID}` is the application ID shown in your app registration.
+1. Sign in with your account.
+1. Review the permissions and click **Accept**.
+
+The browser will attempt to redirect back to your app, which may not be running. You might see a "this site cannot be reached" error after clicking **Accept**. This is OK, the consent was still recorded.
+
+## Run the Sample
```

Comment:
Should there be steps to use the task pane to get user profile information?

Reply:
added